src/main/java/com/bullhornsdk/data/model/entity/core/paybill/JobCode.java

```diff
@@ -0,0 +1,132 @@
+package com.bullhornsdk.data.model.entity.core.paybill;
+
+import com.bullhornsdk.data.api.helper.json.DynamicNullValueFilter;
+import com.bullhornsdk.data.model.entity.core.standard.BusinessSector;
+import com.bullhornsdk.data.model.entity.core.standard.Category;
+import com.bullhornsdk.data.model.entity.core.standard.CorporateUser;
+import com.bullhornsdk.data.model.entity.core.standard.CorporationDepartment;
+import com.bullhornsdk.data.model.entity.core.type.*;
+import com.bullhornsdk.data.model.entity.embedded.OneToMany;
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import lombok.Getter;
+import org.joda.time.DateTime;
+
+import java.math.BigDecimal;
+
+@Data
+@JsonFilter(DynamicNullValueFilter.FILTER_NAME)
```

Comment:
https://stackoverflow.com/questions/56583423/generating-equals-hashcode-implementation-but-without-a-call-to-superclass-even

Could also specify the `@EqualsAndHashCode(callSuper = false)` to avoid the compiler warnings